public/app/features/panel/PanelRenderer.tsx

```diff
@@ -45,24 +45,26 @@ export function PanelRenderer<P extends object = any, F extends object = any>(pr
   const PanelComponent = plugin.panel;
 
   return (
-    <PanelComponent
-      id={1}
-      data={dataWithOverrides}
-      title={title}
-      timeRange={dataWithOverrides.timeRange}
-      timeZone={timeZone}
-      options={optionsWithDefaults!.options}
-      fieldConfig={fieldConfig}
-      transparent={false}
-      width={width}
-      height={height}
-      renderCounter={0}
-      replaceVariables={(str: string) => str}
-      onOptionsChange={onOptionsChange}
-      onFieldConfigChange={setFieldConfig}
-      onChangeTimeRange={onChangeTimeRange}
-      eventBus={appEvents}
-    />
+    <div style={{ width: `${width}px`, height: `${height}px` }}>
```

Comment:
Looks like this is needed for e.g. the table panel.

Reply:
hm... cannot this be handled outside the PanelRenderer? this is already handled by PanelChrome and other places where we use PanelRenderer so would be an unnecessary div in those cases

Reply:
Yes, but my thinking was to add it in the PanelRenderer instead of having to add it on all consumers. But I can move this, no worries :)